system/keymaps/customcontroller.Harmony.xml

```diff
@@ -240,13 +235,6 @@
       <!-- Guide  	-->      <button id="165">NextResolution</button>
     </customcontroller>
   </ScreenCalibration>
-  <ScreenCalibration>
-    <customcontroller name="Harmony">
-      <!-- OK		-->      <button id="105">NextCalibration</button>
```

Comment:
Same here - the global mapping of those buttons is not usefull for the screen calobration. You can't delete them here.

Reply:
I don't get why there needs to be two ScreenCalibration sections, one after the other, mapping almost the same.  But again, there's a lot I don't understand.

Reply:
you are right with this change - i didn't see the second section in the diff ...